src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTagRecordsAndCompactCtors.java

```diff
@@ -1,41 +1,48 @@
 /*
 WriteTag
 tag = @incomplete
-tagFormat = \S
-tagSeverity = (default)info
-tokens = INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF, COMPACT_CTOR_DEF
+tagSeverity = error
+tagFormat = \\S
+tokens = INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF, COMPACT_CTOR_DEF, CTOR_DEF
 
 
 */
 
 //non-compiled with javac: Compilable with Java14
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
-public class InputWriteTagRecordsAndCompactCtors {
+// violation below , 'Type Javadoc comment is missing @incomplete tag'
+public class InputWriteTagRecordsAndCompactCtors { // violation
+
     /**
-     * @incomplete
+     * @incomplete // violation , 'Type Javadoc tag @incomplete must match pattern '\\S''
```

Comment:
Move trailing comment out of javadoc block and use 'violation below'

Reply:
Done. @romani it is this line that is causing the error.